ansible/roles/demo_prep/templates/rds-instance.yml.j2

```diff
@@ -0,0 +1,27 @@
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  name: rds-postgres
+  namespace: "{{ demo_project_name }}"
+spec:
+  serviceClassName: rds-postgres-apb
+  planName: default
+  parameters:
+    aws_access_key: "{{ lookup('env','AWS_ACCESS_KEY_ID') }}"
+    aws_secret_key: "{{ lookup('env','AWS_SECRET_ACCESS_KEY') }}"
+    namespace: "{{ demo_project_name }}"
+    openshift_target: "https://{{ openshift_hostname }}:8443"
+    subnet: "test_awsdemo_rds_group"
```

Comment:
Let's use the variables we have in the secret, they are looked up per account, so when we run in another account it will still work.

  subnet: {{ rds_subnet_group_name }}


https://github.com/fusor/catasb/blob/master/ansible/roles/demo_prep/templates/demo-secret.j2